components/sendgrid/actions/get-contact-lists/get-contact-lists.js

```diff
@@ -0,0 +1,41 @@
+const validate = require("validate.js");
+const common = require("../common");
+
+module.exports = {
+  ...common,
+  key: "sendgrid-get-contact-lists",
+  name: "Get Contact Lists",
+  description: "Allows you to get details of your contact lists.",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    ...common.props,
+    numberOfLists: {
+      type: "integer",
+      label: "Number of Lists",
+      description: "The number of contact lists to return.",
+    },
+  },
+  methods: {
+    ...common.methods,
+  },
+  async run() {
```

Comment:
Include the `$` variable.
```suggestion
  async run({ $ }) {
```